### PR TITLE
NFC - minor spelling tweaks under lite directory

### PR DESCRIPTION
**Pull request by kiszk**
This PR addresses minor spelling tweaks under `tensorflow/lite` directory.
follow-on of #34958

### CODE REVIEW

**Comment by kiszk** [comment]
cc @jaingaurav

File: tensorflow/lite/python/op_hint.py [hunk 1]
```diff
@@ -583,7 +583,7 @@ def aggregate_and_return_name_for_input(self, out_graphdef):
 
     In particular, if you have 4 inputs to a hint stub, this will be the
     node that you can use as an output. I.e. you have 4 timesteps from a
-    static rnn, then a fused UnidriecitonalLSTM will expect 1 input with
+    static rnn, then a fused UndirecitonalLSTM will expect 1 input with
```
Comment (renjie-liu) [inline tensorflow/lite/python/op_hint.py]: UnidirectiontionalLSTM

File: tensorflow/lite/tools/command_line_flags_test.cc [hunk 2]
```diff
@@ -143,7 +143,7 @@ TEST(CommandLineFlagsTest, UsageString) {
   bool some_switch = false;
   std::string some_name = "something";
   // Don't test float in this case, because precision is hard to predict and
-  // match against, and we don't want a flakey test.
+  // match against, and we don't want a franky test.
```
Comment (renjie-liu) [inline tensorflow/lite/tools/command_line_flags_test.cc]: flaky

**Review by renjie-liu — COMMENTED** [review]
thanks a lot for the fix!

**Comment by kiszk** [comment]
@renjie-liu Thank you for pointing them out. I addressed both of them.

File: tensorflow/lite/delegates/gpu/cl/kernels/max_unpooling.cc [hunk 3]
```diff
@@ -25,7 +25,7 @@ namespace gpu {
 namespace cl {
 namespace {
 
-std::string GetMaxUnoolingKernelCode(
+std::string GetMaxUnroolingKernelCode(
```
Comment (renjie-liu) [inline tensorflow/lite/delegates/gpu/cl/kernels/max_unpooling.cc]: should this be unpooling?

File: tensorflow/lite/delegates/gpu/cl/tensor.cc [hunk 4]
```diff
@@ -408,7 +408,7 @@ Status AllocateTensorMemory(const CLContext& context, const CLDevice& device,
     case TensorStorageType::SINGLE_TEXTURE_2D: {
       if (depth != 1) {
         return InvalidArgumentError(absl::StrCat(
-            "SINGLE_TEXTURE_2D support only cnannels in range [1-4], but ",
+            "SINGLE_TEXTURE_2D support only chnannels in range [1-4], but ",
```
Comment (renjie-liu) [inline tensorflow/lite/delegates/gpu/cl/tensor.cc]: channels

File: tensorflow/lite/python/op_hint.py [hunk 5]
```diff
@@ -583,7 +583,7 @@ def aggregate_and_return_name_for_input(self, out_graphdef):
 
     In particular, if you have 4 inputs to a hint stub, this will be the
     node that you can use as an output. I.e. you have 4 timesteps from a
-    static rnn, then a fused UnidriecitonalLSTM will expect 1 input with
+    static rnn, then a fused UndirectionalLSTM will expect 1 input with
```
Comment (renjie-liu) [inline tensorflow/lite/python/op_hint.py]: this does not seem to be changed?

this should be UnidirectionalLSTM

**Review by renjie-liu — COMMENTED** [review]
thanks, looks good! please resolve the inline comments

File: tensorflow/lite/python/op_hint.py (hunk 5 above)
Comment (kiszk) [reply tensorflow/lite/python/op_hint.py]: You are right. sorry for my overlooking.

**Comment by kiszk** [comment]
@renjie-liu Thank you. Addressed your three comments.

File: tensorflow/lite/delegates/gpu/cl/kernels/max_unpooling.cc [hunk 6]
```diff
@@ -131,7 +131,7 @@ MaxUnpooling& MaxUnpooling::operator=(MaxUnpooling&& kernel) {
 }
 
 Status MaxUnpooling::Compile(const CreationContext& creation_context) {
-  const auto code = GetMaxUnoolingKernelCode(
+  const auto code = GetMaxUnroolingKernelCode(
```
Comment (renjie-liu) [inline tensorflow/lite/delegates/gpu/cl/kernels/max_unpooling.cc]: this should be changed as well?

**Review by renjie-liu — DISMISSED** [review]

File: tensorflow/lite/delegates/gpu/cl/kernels/max_unpooling.cc (hunk 6 above)
Comment (kiszk) [reply tensorflow/lite/delegates/gpu/cl/kernels/max_unpooling.cc]: Thank you again, done

**Review by renjie-liu — DISMISSED** [review]

**Comment by gbaned** [comment]
@kiszk Can you please address Ubuntu Sanity errors? Thanks!

**Comment by kiszk** [comment]
Thank you for pinging me. I overlooked pylint error. I have just pushed the fixes.

**Review by renjie-liu — DISMISSED** [review]

**Comment by gbaned** [comment]
@kiszk Can you please resolve conflicts? Thanks!

**Comment by kiszk** [comment]
@gbaned Sure, done

**Review by renjie-liu — DISMISSED** [review]

**Comment by gbaned** [comment]
@kiszk Can you please resolve conflicts? Thanks!

**Comment by kiszk** [comment]
@gbaned sure, done

**Comment by gbaned** [comment]
@kiszk Still, conflicts appearing. Could you please resolve those? Thanks!

**Comment by kiszk** [comment]
@gbaned sure, resolved.

**Comment by kiszk** [comment]
@gbaned Fesolved conflict again.

**Comment by kiszk** [comment]
@gbaned Resolved conflict again.

**Review by renjie-liu — DISMISSED** [review]

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by gbaned** [comment]
@kiszk Can you please resolve conflicts? Thanks!

**Comment by kiszk** [comment]
@gbaned Thank you for pinging me again. Just resolved the conflicts.

**Review by mihaimaruseac — DISMISSED** [review]

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by kiszk** [comment]
@gbaned @mihaimaruseac Resolved a conflict again.

**Review by mihaimaruseac — APPROVED** [review]
Probably it would be better to split this even more given that it keeps getting into conflicts.

Let's give it one more try but if we get new conflicts let's try splitting on the next directory level.

**Comment by mihaimaruseac** [comment]
Manually imported the change and synced to head again. If this still fails to merge I'll suggest splitting it as per previous comment

**Comment by mihaimaruseac** [comment]
Turns out this fails to merge properly. I'm starting a new run to eliminate transient errors but I think it would be better to split it (and resync on master). Apologies for the extra work you'll have to do.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Should be split based on subdirectories of tensorflow/lite

**Comment by gbaned** [comment]
@kiszk Can you please check mihaimaruseac's comments and keep us posted? Thanks!

**Comment by kiszk** [comment]
@gbaned thank you for pinging me. I overlooked the comments. I will split this PR into multiple PRs within next few days
```
tensorflow/lite/
tensorflow/lite/c
tensorflow/lite/delegates
tensorflow/lite/experimental
tensorflow/lite/g3doc
tensorflow/lite/kernels
tensorflow/lite/lib_package
tensorflow/lite/micro
tensorflow/lite/python
tensorflow/lite/testing
tensorflow/lite/toco
tensorflow/lite/tools
```

**Comment by gbaned** [comment]
@kiszk  Sure, Thank you very much for the update.

**Comment by mihaimaruseac** [comment]
You can ping me/assign to me all of the subsequent PRs. Thank you

**Comment by mihaimaruseac** [comment]
Once all subdirectories are fixed, we can sync this back on master to get the files that are left out. Or, we can just close it now and get the other PRs as needed.

Thank you for all the fixes.

**Comment by kiszk** [comment]
I created all of sub-PRs. When they are closed, I think that it would be good to close this PR, too.

**Comment by mihaimaruseac** [comment]
Sounds good. Thank you

**Comment by mihaimaruseac** [comment]
Everything seems solved. Let's rebase this on master if there is still work left to do or close otherwise.

Thank you

**Comment by kiszk** [comment]
@mihaimaruseac Thank you very much. It is the time to close this.

**Comment by mihaimaruseac** [comment]
Thank you for all the work and for the patience to carry on these PRs over 4 months

**Pull request closed without merging**